atom/browser/atom_browser_context.cc

```diff
@@ -37,14 +58,25 @@ std::string RemoveWhitespace(const std::string& str) {
     return str;
 }
 
+// Convert string to lower case and escape it.
+std::string MakePartitionName(const std::string& input) {
+  return net::EscapePath(base::ToLowerASCII(input));
+}
+
 }  // namespace
 
+// static
+AtomBrowserContext::BrowserContextMap AtomBrowserContext::browser_context_map_;
```

Comment:
I thought Chromium's linting didn't allow items with complex destructors to be declared as globals like this...?

Not a blocker, but asking because I'm not sure how this got past the linter or if I'm misunderstanding :smile:

Reply:
I believe the linter only checks for static/global strings.